dgfit/plotting/plot_dgfit.py

```diff
@@ -279,6 +284,7 @@ def plot_dgfit_g(
 
     ax.set_xlim(get_krange(hdu.data["WAVE"], logaxis=True))
     ax.set_ylim([0.0, 1.0])
+    ax.set_title("Scattering phase funtion")
```

Comment:
Are titles to the plots needed?  Do the contents of the plots (e.g., x-axis labels) not telling one what the plots are?  Asking as in general journals prefer not to have titles to plots (at least they did in the past).